plugin/proto/metadata.go

```diff
@@ -13,40 +23,40 @@ type Metadata struct {
 }
 
 // Validate checks if the metadata is correctly populated.
-func (m *Metadata) Validate() error {
-	if m.Name == "" {
+func (resp *GetMetadataResponse) Validate() error {
+	if resp.Name == "" {
 		return errors.New("empty name")
 	}
-	if m.Description == "" {
+	if resp.Description == "" {
 		return errors.New("empty description")
 	}
-	if m.Version == "" {
+	if resp.Version == "" {
 		return errors.New("empty version")
 	}
-	if m.URL == "" {
+	if resp.URL == "" {
 		return errors.New("empty url")
 	}
-	if len(m.Capabilities) == 0 {
+	if len(resp.Capabilities) == 0 {
 		return errors.New("empty capabilities")
 	}
-	if len(m.SupportedContractVersions) == 0 {
+	if len(resp.SupportedContractVersions) == 0 {
 		return errors.New("empty supported contract versions")
 	}
 	return nil
 }
 
-func (Metadata) Command() Command {
+func (GetMetadataResponse) Command() Command {
 	return CommandGetMetadata
 }
```

Comment:
Do we need this?

`Command()` is not present on other Response structs. Let's be consistent across all plugin requests/responses.

Reply:
Yes, we need Command() for request to generate the command name for calling the plugin. Response is only a struct for parsing the json output.

Reply:
Why do we have `Command()` on GetMetadataResponse here?

Reply:
Thanks. We don't need it. Removed.